### ISSUE TITLE
How to use **onMessagesSeen** event to mark messages as read on my database

### ISSUE THREAD

**Issue by rpaschoal** [rpaschoal/ng-chat#75]
Moving this to a new issue as we can discuss it further here and within context:

How i can get call when onMessagesSeen event triggered. how to implement this in my adapter or component, any sample code.(i want to mark them read in my database).
thanks in advance.

_Originally posted by @MuhammadRizwanBashir in https://github.com/rpaschoal/ng-chat/issues/72#issuecomment-429624551_

**Comment by rpaschoal** [comment]
Some instructions on this shared by @FShieldheart on #72 

Here what I did in my project. I wrote a service and injected into my component which has the "onMessagesSeen" method. This service uses an API to persist unread messages as read in the data store. Also, the same API sends a signal to the receiver via SignalR upon successful completion of the transaction so that sender is notified of the messages which have read (assuming sender is online at the time of this flow). Hope it helps.
